service/awsclusterconfig/v1/version_bundle.go

```diff
@@ -0,0 +1,20 @@
+package v1
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func VersionBundle() versionbundle.Bundle {
+	return versionbundle.Bundle{
+		Changelogs:   []versionbundle.Changelog{},
+		Components:   []versionbundle.Component{},
+		Dependencies: []versionbundle.Dependency{},
```

Comment:
Should there be dependencies for other operators (e.g. minimum version of supported `aws-operator` is this and this)?

Reply:
Lets keep it empty for now. We can restrict this as soon as it makes sense.